nixos/modules/misc/ids.nix

```diff
@@ -576,6 +577,7 @@
       mighttpd2 = 285;
       hass = 286;
       monero = 287;
+      morty = 288;
```

Comment:
Why does this need a static uid?

Reply:
I'm new at packaging services, and basically replicated the setup of searx which is its companion tool. That set uid and gid, so I did that for morty as well.
 
Is there a checklist for adding services somewhere?

Reply:
Not really; I'm just wondering whether there's a reason for allocating a uid; if there's a reason that's totally fine, otherwise you can probably get away with dynamic uids.

Reply:
I'm certainly not an expert in this. If you want this removed, let me know.

Reply:
What type of data does the service generate? Are you likely to transfer it across hosts & is it too expensive to fixup permissions at service start or do you need to refer to the uid before the uid/name mapping is established? If so, using a static uid may be appropriate. I'm happy to defer to your judgement regarding the service, I know nothing about it, but often static uids are used for no good reason and so the pattern propagates.